.travis.yml

```diff
@@ -99,7 +99,7 @@ matrix:
     - env: TEST_SUITES=testbugfix CONFIGFLAGS="--enable-debug"
 
     # test error reporting and compiling (quickest job in this test suite)
-    - env: TEST_SUITES=testspecial CONFIGFLAGS="--enable-debug"
+    - env: TEST_SUITES="testspecial test-compile" CONFIGFLAGS="--enable-debug"
```

Comment:
Hmm, am I missing something? The description says "split the Travis test suite into two: one which just does testspecial, and one which does just test-compile" but it runs them in a single job.

Reply:
test suite != test job. The test suite was split into two; but both are still run by the same job (this is also explained in the commit message).